distributed/comm/core.py

```diff
@@ -0,0 +1,292 @@
+from __future__ import print_function, division, absolute_import
+
+from abc import ABCMeta, abstractmethod, abstractproperty
+from datetime import timedelta
+import logging
+
+from six import string_types, with_metaclass
+
+from tornado import gen
+from tornado.ioloop import IOLoop
+
+from ..metrics import time
+
+
+logger = logging.getLogger(__name__)
+
+# Connector instances
+
+connectors = {
+    #'tcp': ...,
+    #'zmq': ...,
+    }
+
+
+# Listener classes
+
+listeners = {
+    #'tcp': ...,
+    # 'zmq': ...,
+    }
+
+
+DEFAULT_SCHEME = 'tcp'
```

Comment:
We might want to pull this value from `config`.  This would make it easier to switch to ZeroMQ without changing source.